stripes.config.js

```diff
@@ -13,12 +13,13 @@ module.exports = {
   },
   modules: {
     '@folio/agreements': {},
+    '@folio/erm-usage': {},
     '@folio/licenses': {},
     '@folio/orders': {},
     '@folio/plugin-find-agreement': {},
     '@folio/plugin-find-license': {},
     '@folio/plugin-find-user': {},
-    '@folio/stripes-erm-components': {},
```

Comment:
Can you confirm that removing `stripes-erm-components` this was intended? Without having it in the modules list, Stripes won't handle the translation files for it.

Reply:
Intended - yes.  Checked - no :) 

The issue I was having was that stripes-cli was generating an empty MD which also doesn't exist on http://folio-registry.aws.indexdata.com which subsequently caused some okapi dependency checks to fail in CI.   So I removed it assuming it functioned similarly to stripes-components.   However,  I will add it back to stripes.config.js and enable posting of an (empty) MD to the folio-registry as a workaround.

Reply:
Actually,  a better solution is to just fix it in CI.  So that's what I'l do.   I did also notice that stripes-erm-components is not listed in package.json.   Should it be listed as a dep there and pinned to ~1.2.1?

Reply:
stripes-erm-components is back in stripes.config.js and "pinned" as '~1.2.1' in package.json (as discussed on slack")